docs/discussion/concepts/index.md

```diff
@@ -26,6 +31,10 @@ and help you use units libraries more effectively.
 - **[Quantity Point](./quantity_point.md)**.  An abstraction for "point types" that have units.
   Most use cases don't need this, but for a few --- including temperatures --- it's indispensable.
 
+- **[Truncation](./truncation.md)**.  Unit conversions truncate when the destination type can't hold
+  the result with acceptable precision.  This page will explain Au's approach to managing this risk,
```

Comment:
```suggestion
  the result with acceptable precision.  This page explains Au's approach to managing this risk,
```

Reply:
This is pretty funny.  When I wrote this entry, I hadn't yet written the truncation page.  So I was thinking of this in my head as future work, and it accidentally slipped into my phrasing!